Sources/NIOHTTP2/ConnectionStateMachine/ConnectionStateMachine.swift

```diff
@@ -1042,7 +1042,7 @@ extension HTTP2ConnectionStateMachine {
             return .init(result: .connectionError(underlyingError: NIOHTTP2Errors.missingPreface(), type: .protocolError), effect: nil)
 
         case .fullyQuiesced:
-            return .init(result: .connectionError(underlyingError: NIOHTTP2Errors.ioOnClosedConnection(), type: .protocolError), effect: nil)
+            return .init(result: .succeed, effect: nil)
```

Comment:
Can you add a brief code comment on why we allow RST_STREAM frames to be sent when in this state?